File: Basics/Kotlin/app/src/main/java/com/suavepirate/kotlinbasics/samples/constructors/Dog.kt
package com.suavepirate.kotlinbasics.samples.constructors


class Dog() {
    constructor(name: String?, age: Int?, breed: String?) : this() {
        this.name = name
        this.age = age
        this.breed = breed
    }
    var name: String? = null
    var age: Int? = null
    var breed: String? = null
}

File: Basics/C#/CSharp.Basics/Samples/Constructors/Dog.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace CSharp.Basics.Samples.Constructors
{
    public class Dog
    {
        public string Name { get; set; }
        public int Age { get; set; }
        public string Breed { get; set; }

        public Dog()
        {

        }

        public Dog(string name, int age, string breed)
        {
            Name = name;
            Age = age;
            Breed = breed;
        }
    }
}
